docs/embedded/concepts/app-concepts/containertypes.md

```diff
@@ -1,47 +1,92 @@
 ---
-title: Container Type
+title: SharePoint Embedded Container Types
 description: This article explains how Container Types work.
-ms.date: 11/28/2023
+ms.date: 03/05/2023
 ms.localizationpriority: high
 ---
-
+ 
 # SharePoint Embedded Container Types
 
-In SharePoint Embedded, all files and documents are stored in Containers, and each Container is identified by a Container Type.
+In SharePoint Embedded, files and documents are stored in Containers, each identified by a Container Type, which is a property stamped on every instance. A Container Type links all Containers to a specific Azure Subscription for billing. Each Container Type is owned by a single SharePoint Embedded Application, and each SharePoint Embedded Application can own only one Container Type. It specifies the access permissions an SharePoint Embedded Application has for all Containers of that type, such as creating, reading, writing, deleting containers, and managing container permissions.
 
-Container Type is a property stamped on every Container instance. Each Container Type is owned by one Application; and each Application can own only one Container Type.
+## Two Kinds of Container Types
+There are two kinds of Container Types that a developer tenant can create:
+1. [Trial Container Types](#trial-container-types)
+2. [Standard Container Types](#standard-container-types)
```

Comment:
Markdown
- numbered bullets should always be `1.`... don't use explicit numbers... let the rendering engine take care of that
- headings, paragraphs, and bullet lists should have line breast around them (there should be empty lines after lines 12 & 13)